### PR TITLE
Depend on: `prefix` -> `opt_prefix`

### PR DESCRIPTION
**Pull request by tamird**

### CODE REVIEW

**Comment by jacknagel** [comment]
Use `opt_prefix` instead

File: Library/Formula/gearman.rb [hunk 1]
```diff
@@ -107,7 +107,7 @@ def plist; <<-EOS.undent
 +++ b/libgearman-1.0/gearman.h
 @@ -50,7 +50,11 @@
  #endif
- 
+
```
Comment (jacknagel) [inline Library/Formula/gearman.rb]: don't change whitespace in patches, please

Comment (tamird) [inline Library/Formula/gearman.rb]: it was in a separate commit - still no good?

Comment (jacknagel) [inline Library/Formula/gearman.rb]: whitespace is significant in patches (the first column is always either a space, a `+`, or a `-`, and trailing whitespace is present in the source lines which provide context for the patch)

Comment (tamird) [inline Library/Formula/gearman.rb]: oh, that's what you mean. understood.

**Comment by tamird** [comment]
Changed to `opt_prefix`. Thanks!

**Pull request closed without merging**